### PR TITLE
[script.simkl] v1.0.4

### PR DESCRIPTION
**Pull request by daviddavo**
### Description

Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into.
### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.simkl/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.4"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
```
Comment (enen92) [inline script.simkl/addon.xml]: xbmc.python version should be set at 2.24.0 to match jarvis. See: http://kodi.wiki/view/addon.xml

File: script.simkl/resources/lib/engine.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+__addon__ = interface.__addon__
+def getstr(strid): return interface.getstr(strid)
+
+class Engine:
+  def __init__(self, api, player):
+    self.api = api
+    self.player = player
+    player.engine = self
+    player.api    = api
+    self.synclibrary()
```
Comment (enen92) [inline script.simkl/resources/lib/engine.py]: probably you can remove this call

**Comment by enen92** [comment]
Looks okay to me, please adjust the python version (and the second comment which is not mandatory) so it can be merged

File: script.simkl/addon.xml (hunk 1 above)
Comment (daviddavo) [inline script.simkl/addon.xml]: You mean 2.14?

**Pull request merged**